Source/SwiftLintFramework/Extensions/QueuedPrint.swift

```diff
@@ -34,9 +34,12 @@ public func queuedPrint<T>(object: T) {
 A thread-safe, newline-terminated version of fputs(..., stderr).
 
 - parameter string: String to print.
+- parameter silent: True suppresses output
 */
-public func queuedPrintError(string: String) {
-    dispatch_async(outputQueue) {
-        fputs(string + "\n", stderr)
+public func queuedPrintError(string: String, silent: Bool = false) {
```

Comment:
Sure, but because the new code results in a new configuration load for every file under a nested configuration, reverting this will result in dozens of "Loading configuration from " messages when using nested configs. I found this really obfuscates the logs. Another option would be to suppress the logging inside the `init` function, but that didn't seem right to me. My intention here was to centralize the logging logic in one place so that the logger could decide how to process. Because we wouldn't have to hunt down selective logging in methods, this could potentially ease the transition to more robust logging if needed. So let me know which of the three solutions you'd like:
 1. Revert to original, which is the simplest but logs will be muddled with new load messages
 2. Put the selective logging logic in the `init` function which requires no change to logging functions, but makes `init` more complicated and sets the precedent of distributing logging logic to non-logging objects.
 3. Put the selective logging logic in the logger which solves the above problems but results in a degenerative trivial silent logging case